app/templates/ember_charts/time_series.hbs

```diff
@@ -60,6 +61,7 @@
              contentBinding="tickIntervals"
              valueBinding="selectedInterval"
              class="form-control"
+             disabled=dynamicXAxis
```

Comment:
The tick interval property is only relevant when it is not a dynamic x axis. When it is a dynamic x axis, the tick interval is essentially set by the internal logic. When it is not a dynamic x axis, the tick interval defines the x axis labelling.